guides/4.contexts.rst

```diff
@@ -341,3 +341,17 @@ And then use it in your context:
 
 It will just work as you expect it to.
 
+Context traits come in handy if you'd like to have separate contexts,
+but still need use the very same step definition in both of them. Instead of
+having the same code in both context classes – and having to maintain it
+in both – you should create a single Trait that you would then ``use`` in
+both context classes.
+
+.. note::
+
+    Given that step definitions :doc:`cannot be duplicated within a Suite </guides/2.definitions>`,
+    this will only work for contexts used in separate suites.
+
+    In other words, if your Suite uses at least two different Contexts, and
+    those context classes ``use`` the same Trait, this will result in a duplicate
+    step definition and behat will complain by throwing a `Redundant`` exception.
```

Comment:
On 17/01/15 02:09, Christophe Coevoet wrote:
>
> there is a markup issue here (backticks are not consistent)
>
Uugh, sorry. I should probably preview my text before committing. Have
looked for a preview tool and found rst.ninjs.org.